orc8r/tools/packer/scripts/upgrade-ovs-patch-1.1.20210618.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+set -x
+
+export version="2.15.4-9-magma"
+export DEBIAN_FRONTEND="noninteractive"
+
+sudo rm -rf /etc/apt/sources.list.d/*.list
+sudo rm -rf /var/cache/apt/
+
+echo "Installing certificates ..."
+sudo apt-get update
+sudo apt-get install -y ca-certificates
```

Comment:
Is this needed to add the third party repository?
I not, we should move it below in order to avoid one of the `apt update` calls.

Reply:
yes, it is needed